bin/phase-helpers.sh

```diff
@@ -544,7 +544,7 @@ unpack() {
 						"with EAPI '${EAPI}'. Instead use 'xz'."
 				fi
 				if ___eapi_unpack_supports_xz; then
-					__unpack_tar "xz -d"
+					__unpack_tar "xz -T$(___makeopts_jobs) -d"
```

Comment:
If MAKEOPTS does not contain a `-j` flag, `___makeopts_jobs` outputs `-1`. Does xz handle this appropriately?

Reply:
Is it not 1?

Reply:
With MAKEOPTS="", I get:
````
portage   560660  1.0  0.0  14628  5796 pts/27   S+   22:53   0:00 xz -T1 -d -c -- /var/tmp/portage/dev-lang/rust-1.64.0/distdir/rustc-1.64.0-src.tar.xz
````

This seems to be working correctly.

Reply:
My mistake, nevermind.